libc-top-half/musl/src/thread/pthread_create.c

```diff
@@ -364,6 +450,26 @@ int __pthread_create(pthread_t *restrict res, const pthread_attr_t *restrict att
 		if (ret)
 			__wait(&args->control, 0, 3, 0);
 	}
+#else
+	/* As in the unmodified version, all spawn failures translate to EAGAIN.
+	 * If the spawn did succeed, then wait for the TID to be populated by the
+	 * child thread. */
+	if (ret != 0) {
+		ret = -EAGAIN;
+	} else {
+		int timeout_ns = -1; // TODO
```

Comment:
How long to wait here for the child thread to report it's TID?

Reply:
I don't see why we would just wait forever.   If the thread doesn't start that would be bug in the hos wouldn't it?  I don't think need to worry such things at runtime.

Reply:
You mean "I don't see why we would _not_ just wait forever"?

Reply:
Yes sorry!

Reply:
Yeah, I guess I'll just document the "wait forever... bug in the host" bit and leave it at `-1`.